app/models/communities.rb

```diff
@@ -0,0 +1,59 @@
+class Communities
+  require "yaml"
+
+  @local_chapters = {}
+
+  def self.local_chapters(locale)
+    @local_chapters[locale] = local_chapter_for(locale)
+  end
+
+  class << self
+    protected
```

Comment:
Is this the reason for the black magic? Because you can't have a protected class method so you'd making them instance methods on the metaclass instead? Why do you want protected anyway - is something going to inherit from this?